analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd

```diff
@@ -224,6 +255,54 @@ histologies_df %>%
   write_tsv(output_file)
 ```
 
+### Methylation embryonal subtypes
+
+Write to file high confidence methylation classification subtypes
+
+```{r}
+# high confidence methylation classification subtypes
+emb_methyl_subtyped <- histologies_df %>% 
+  dplyr::filter(pathology_diagnosis %in% path_dx_list$include_path_dx |
+                  pathology_free_text_diagnosis %in% path_dx_list$include_free_text,
+                dkfz_v12_methylation_subclass_score >= 0.8,
+                experimental_strategy == "Methylation", 
+                sample_type == "Tumor", 
+                composition == "Solid Tissue",
+                cohort != "TARGET") %>% 
+  dplyr::mutate(molecular_subtype_methyl = 
+                  case_when(dkfz_v12_methylation_subclass == "CNS_NB_FOXR2" ~ "CNS NB-FOXR2",
+                            dkfz_v12_methylation_subclass == "ETMR_C19MC" ~ "ETMR, C19MC-altered",
+                            dkfz_v12_methylation_subclass == "CNS_BCOR_ITD" ~ "CNS BCOR",
+                            dkfz_v12_methylation_subclass == "NET_CXXC5" ~ "CNS HGNET-MN1",
+                            dkfz_v12_methylation_subclass == "ABM_MN1" ~ "CNS HGNET-MN1")) %>% 
+  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, molecular_subtype_methyl) %>%
+  dplyr::rename("Kids_First_Biospecimen_ID_Methyl" = "Kids_First_Biospecimen_ID") %>% 
+  dplyr::filter(!is.na(molecular_subtype_methyl)) %>% 
+  dplyr::distinct()
+  
+# low confidence methylation classification subtypes
+emb_methyl_not_subtyped <- histologies_df %>% 
+  dplyr::filter(pathology_diagnosis %in% path_dx_list$include_path_dx |
+                  pathology_free_text_diagnosis %in% path_dx_list$include_free_text,
+                !Kids_First_Biospecimen_ID %in% unique(emb_methyl_subtyped$Kids_First_Biospecimen_ID_Methyl),
+                experimental_strategy == "Methylation", 
+                sample_type == "Tumor", 
+                composition == "Solid Tissue",
+                cohort != "TARGET") %>%
+  dplyr::left_join(emb_methyl_subtyped %>% dplyr::select(Kids_First_Participant_ID, sample_id, molecular_subtype_methyl),
+                   by = c("Kids_First_Participant_ID", "sample_id")) %>%  
+  dplyr::mutate(molecular_subtype_methyl = case_when(
+    is.na(molecular_subtype_methyl) ~ "CNS Embryonal, NOS",
+    TRUE ~ molecular_subtype_methyl)) %>% 
```

Comment:
```suggestion
  dplyr::mutate(molecular_subtype_methyl = case_when(
    is.na(molecular_subtype_methyl) ~ NA_character,
    TRUE ~ molecular_subtype_methyl)) %>% 
```

Reply:
We don't want to create new methylation subtypes - only report high confidence or none